src/python/zquantum/qcbm/ansatz.py

```diff
@@ -1,7 +1,21 @@
 import numpy as np
 from zquantum.core.circuit import Circuit, Qubit, Gate
 
-def get_single_qubit_layer(n_qubits, params, single_qubit_gates):
+def get_qcbm_ansatz(n_qubits, n_layers, topology):
+    """
+    TODO
+    """
+    ansatz = {'ansatz_type': 'QCMB_ion_trap',
```

Comment:
Caught a typo here: should be QCBM and not QCMB. 

Also, some time ago I discussed with @max-radin about how to store ansatz information and I think we didn't get to a final resolution. Is there one now?

Reply:
Thanks!
We will introduce an interface for ansatzes soon, but needed to get this done first :)

Reply:
Adressed in `0203b0f7888b4418afac825051d323b2351734da`.